src/components/CategoriesList.js

```diff
@@ -8,7 +8,7 @@ export const CategoriesList = (props) => {
   return (
     <div className="Categories-list"> 
       {
-        categories.map(category => <Category name={category.name} image={category.image}/>) 
+        categories.map(category => <Category key={category.name} name={category.name}/>) 
```

Comment:
The key is a react thing, it will give you a warning if you don't add a key to the element in a map.